### PR TITLE
Overlay for external video with volume controller and reload button

### PR DESCRIPTION
**Pull request by lfzawacki**
### What does this PR do?

Adds two components that are rendered over the external video player:
  * A generic volume slider that works for most react players that support it.
  * A reload button that will refresh the current video. This is useful for users experiencing temporary YouTube errors or needing to reload the video because the presenter has changed the permissions.

Both buttons show up for viewers and not presenters, since the presenter will have full access to the player controls and can reload the video by simply sharing the video again.

### Motivation

This allows us to hide all the external video controls for viewers (when the external player allows it) and still present the user with a custom volume slider that enables volume control. The reload button 

### More

Currently we have only one volume icon called 'speak_louder', but ideally we should have a few different ones to show low volume, normal volume and muted states on the volume slider.

### CODE REVIEW

**Comment by lfzawacki** [comment]
Should begin to address https://github.com/bigbluebutton/bigbluebutton/issues/10640 and https://github.com/bigbluebutton/bigbluebutton/issues/7652

File: bigbluebutton-html5/client/stylesheets/bbb-icons.css [hunk 1]
```diff
@@ -160,6 +160,22 @@
...
+  content: "\e809";
+}
+
+.icon-bbb-volume_off:before {
+  content: "\e80a";
+}
+
+.icon-bbb-volume_up:before {
+  content: "\e80b";
+}
```
Comment (pedrobmarin) [inline bigbluebutton-html5/client/stylesheets/bbb-icons.css:177]:
```suggestion
.icon-bbb-volume_down:before {
  content: "\e947";
}

.icon-bbb-volume_mute:before {
  content: "\e947";
}

.icon-bbb-volume_off:before {
  content: "\e947";
}

.icon-bbb-volume_up:before {
  content: "\e947";
}
```

**Comment by pedrobmarin** [comment]
❤️

**Comment by antobinary** [comment]
Nice @lfzawacki !
Did you mean to send the PR towards 'develop' to be possibly included in 2.3?

**Comment by lfzawacki** [comment]
I've only tested this based on 2.2, it should work with 2.3, BUT I'll mark this as a draft so that we can test it better after the 2.3 release.

**Comment by lfzawacki** [comment]
Pointing base to 'develop' and fixing merge conflicts. Everything seems to be working, we're just missing some different icons.

**Comment by antobinary** [comment]
@jfsiebel and myself tested this pull request and our findings are listed in https://github.com/bigbluebutton/bigbluebutton/issues/12802 - they can be addressed separately from this pull request.

I also don't see the volume control but it might be under the refresh button.
I also don't see the new buttons when I am the presenter.

In the interest of proceeding with this work, we'll have someone else take it from here.
Thanks @lfzawacki

**Pull request merged**